doc/modules.rst

```diff
@@ -101,6 +101,12 @@ policies (governors). The ``devlib`` module exposes the following interface
    :param cpu: The cpu; could be a numeric or the corresponding string (e.g.
        ``1`` or ``"cpu1"``).
 
+.. method:: target.cpufreq.get_min_available_frequency(cpu)
+            target.cpufreq.get_max_available_frequency(cpu)
+
+    Retrieve the the minimum or maximum DVFS frequency available for the
+    specified CPU. Returns an int or None if could not be determined.
```

Comment:
This should probably go below ``get_<min|max>_frequency`` and the description should clarify how these are different.

Reply:
Updated.